src/js/types.ts

```diff
@@ -238,8 +238,13 @@ export interface Module {
   noWasmDecoding: boolean;
   quit: (status: number, toThrow: Error) => void;
   preRun: { (): void }[];
-  print: (a: string) => void;
-  printErr: (a: string) => void;
+  print?: (a: string) => void;
+  printErr?: (a: string) => void;
+  arguments: string[];
+  API: API;
```

Comment:
Maybe we need to separate methods that comes from emscripten and methods that are added by Pyodide ( no action required for this PR ).

Reply:
Also, we should move away from Pyodide adding things as much as possible.